go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/feiguoL/uniplot
+
+go 1.17
+
+require (
+	github.com/aybabtme/uniplot v0.0.0-20151203143629-039c559e5e7e
```

Comment:
```suggestion
```
don't need this if you keep the `module` name as `github.com/aybabtme/uniplot`

Reply:
done in new pull request!